packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -109,7 +109,7 @@ Future<List<Plugin>> findPlugins(FlutterProject project, { bool throwOnError = t
 /// Plugin type to determine the injection mechanism.
 enum _PluginType {
```

Comment:
`nativeOrDart` doesn't seem like a plugin type, so this should probably be called `_PluginResolutionType`, and the comment adjusted accordingly.

Reply:
Done in d621a904e4a2ae069a1d02926ce7142553ff6fe4